packages/liveblocks-react-lexical/src/comments/comment-plugin-provider.tsx

```diff
@@ -55,6 +52,15 @@ export const ThreadToNodesContext = createContext<ThreadToNodesMap | null>(
   null
 );
 
+function useCommentsErrorListener(callback: (error: LiveblocksError) => void) {
+  const client = useClient();
+  const savedCallback = useLatest(callback);
+
+  useEffect(() => {
+    return client.events.error.subscribe(savedCallback.current);
+  }, [client, savedCallback]);
+}
```

Comment:
Isn't this the same as the new `useErrorListener`?

Reply:
Thanks for catching this. I DRYed this up with the normal listener in 5ede3f82761433277ef7124b10abf428949b8818 👍

Reply:
Actually… I think with https://github.com/liveblocks/liveblocks/pull/2167#issuecomment-2589615909 now in place, this will actually trigger this DX warning when "just" using our Lexical integration now 🤔 Also it will silence [these dev warnings](https://github.com/liveblocks/liveblocks/pull/2167/commits/e0a0a9396fd6396ee46bfce94a134ad54140e83f) by default now when you use our Lexical package 🤔

Reply:
Going to revert [this change](https://github.com/liveblocks/liveblocks/pull/2167#issuecomment-2589615909) to avoid this issue.

Reply:
Done in ef3f1e7c89f56fda06e8974dc0aaa0f846cab547.